clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/provider/agent/TaskHealthCachingAgentSpec.groovy

```diff
@@ -146,6 +159,17 @@ class TaskHealthCachingAgentSpec extends Specification {
     def taskCacheData = new DefaultCacheData(taskKey, taskAttributes, Collections.emptyMap())
     providerCache.getAll(TASKS.toString(), _) >> Collections.singletonList(taskCacheData)
 
+    Map<String, Object> containerDefinitionMap = mapper.convertValue(new ContainerDefinition().withPortMappings(
+      new PortMapping().withHostPort(1338)
+    ), Map.class)
```

Comment:
Is there a test in here that tests the dynamic port mapping case?  i.e. host port is 0 in the task def to ensure we don't attempt to use that value when querying task health

Reply:
Added.